diff --git a/node_modules/icomoon-generator/dist/cjs/index.js b/node_modules/icomoon-generator/dist/cjs/index.js
index 9a1195e..6c6da2a 100644
--- a/node_modules/icomoon-generator/dist/cjs/index.js
+++ b/node_modules/icomoon-generator/dist/cjs/index.js
@@ -31,8 +31,9 @@ var PAGE = {
   FIRST_ICON_BOX: "#set0 .miBox:not(.mi-selected)",
   REMOVE_SET_BUTTON: ".menuList2.menuList3 li:last-child button",
   SELECT_ALL_BUTTON: 'button[ng-click="selectAllNone($index, true)"]',
-  GENERATE_LINK: 'a[href="#/select/font"]',
-  GLYPH_SET: "#glyphSet0",
+  // GENERATE_LINK: 'a[href="#/select/font"]',
+  GENERATE_LINK: 'a[href="#/select/image"]',
+  GLYPH_SET: "#glyphSet0, #set0",
   GLYPH_NAME: ".glyphName",
   DOWNLOAD_BUTTON: ".btn4"
 };
@@ -396,9 +397,13 @@ function _pipeline() {
             }, names);
 
           case 86:
-            _context3.next = 88;
+            _context3.next = 87;
             return sleep(1000);
             
+          case 87:
+            _context3.next = 88;
+            return page.click(PAGE.GENERATE_LINK).catch(()=>{});
+
           case 88:
             _context3.next = 90;
             return page.reload();
@@ -413,7 +418,8 @@ function _pipeline() {
 
           case 94:
             meta = selection.preferences.fontPref.metadata;
-            zipName = meta.majorVersion ? meta.fontFamily + "-v" + meta.majorVersion + "." + (meta.minorVersion || 0) + ".zip" : meta.fontFamily + ".zip";
+            // zipName = meta.majorVersion ? meta.fontFamily + "-v" + meta.majorVersion + "." + (meta.minorVersion || 0) + ".zip" : meta.fontFamily + ".zip";
+            zipName = "icomoon.zip";
             logger("Started to download " + zipName);
             zipPath = path.join(outputDir, zipName);
             _context3.next = 100;
@@ -444,7 +450,8 @@ function _pipeline() {
                         break;
                       }
 
-                      _context2.next = 6;
+                      // _context2.next = 6;
+                      _context2.next = 12;
                       return whenFinished({
                         outputDir: outputDir
                       });
